index.html

```diff
@@ -3563,7 +3563,7 @@ <h3><dfn>Writing or pushing content</dfn></h3>
                 Let |matched:boolean| be `false`.
               </li>
               <li>
-                If |reader|.<a>[[\Id]]</a> is not empty string and is equal to
+                If |reader|.<a>[[\Id]]</a> is not `undefined` and is equal to
                 |record|'s <a href="#dom-ndefrecord-id">id</a>,
                 [= iteration/continue =].
                 If it is equal, set |matched| to `true`.
```

Comment:
If `Id` is `undefined`, means it matches anything, we'll continue to look at other fields.